frontend/src/components/log-in/Login.jsx

```diff
@@ -5,20 +5,53 @@ import logo from "../../assets/UnstractLogoBlack.svg";
 import loginRightBanner from "../../assets/login-right-panel.svg";
 import { getBaseUrl } from "../../helpers/GetStaticData";
 import "./Login.css";
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
 
 let LoginForm = null;
+let PRODUCT_NAMES = {};
+let SELECTED_PRODUCT;
+let selectedProductStore;
+let setSelectedProduct;
 try {
   LoginForm = require("../../plugins/login-form/LoginForm").LoginForm;
+  PRODUCT_NAMES = require("../../plugins/llm-whisperer/helper").PRODUCT_NAMES;
+  SELECTED_PRODUCT =
+    require("../../plugins/llm-whisperer/helper").SELECTED_PRODUCT;
+  selectedProductStore = require("../../plugins/llm-whisperer/store/select-product-store.js");
 } catch {
   // The components will remain null of it is not available
 }
+
 function Login() {
+  const location = useLocation();
+  const queryParams = new URLSearchParams(location.search);
+  const selectedProduct = queryParams.get(SELECTED_PRODUCT);
```

Comment:
@tahierhussain Since this a plugin, will the below part fail if  SELECTED_PRODUCT is None?